docker/compose/withPostgres/docker-compose.yml

```diff
@@ -39,7 +39,6 @@ services:
       - postgres
     volumes:
       - n8n_storage:/home/node/.n8n
-    command: /bin/sh -c "n8n start --tunnel"
```

Comment:
We're losing the `--tunnel` capability, aren't we?

Reply:
our tunnel servers seem overwhelmed. I haven't managed to get them working for a while.

Reply:
Ah damn, maybe we should switch to something like ngrok if it's possible instead of trying to host it ourselves.

Reply:
it could be that's `localtunnel` is simply not working for me. I can put the `--tunnel` option back if you prefer. but I think people running either of these docker-compose stacks need a tunnel url. I always assumed that was for user running n8n via `npx`.

Reply:
I just tried starting n8n with `--tunnel` and I get a URL but it won't work to load n8n. So I think we should continue with the `--tunnel` flag, and tell users that we're having issues maintaining this service rather than just removing it without prior notice WDYT @netroy ?

Reply:
I'm working on bringing this back, once the app uses the tunnel url correctly throughout.